### ISSUE TITLE
pre-trained weights

### ISSUE THREAD

**Issue by florencejt** [jamespengcheng/PyTorch-CNN-on-CIFAR10#1]
Hi, this looks great! Do you have the pretrained weights that you could upload please?
Thank you :)

**Comment by Abderrahmenee** [comment]
You can train it yourself. use `load_pretrained_model = False`

You might also need to change code in line 117 with `running_loss += loss.data.item()` instead of `running_loss += loss.data[0]`
